src/sentry/integrations/models/data_forwarding_project.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import annotations
+
+from django.db import models
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import DefaultFieldsModel, FlexibleForeignKey, region_silo_model
+from sentry.db.models.fields.jsonfield import JSONField
+
+
+@region_silo_model
+class DataForwardingProject(DefaultFieldsModel):
+    """
+    Links DataForwardingConfig to specific projects with optional project-specific overrides.
+    """
+
+    __relocation_scope__ = RelocationScope.Organization
+
+    config = FlexibleForeignKey(
+        "sentry.DataForwardingConfig",
+        on_delete=models.CASCADE,
+        related_name="project_configs",
```

Comment:
I think `projects` makes sense as a shorter related name, unless it can't be used for some reason.